docs/content/en/docs/user-guide/configuring-deployment/ecs.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "ECS"
+linkTitle: "ECS"
+weight: 5
+description: >
+  Specific guide for configuring Amazon ECS deployment.
+---
+
+Deploying a Amazon ECS application requires `taskdef.yaml` and `servicedef.yaml` file placing inside the application directory. Those files contain all configuration for [ECS TaskDefinition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html) object and [ECS Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html) object, and will be used by Piped agent while deploy your application/service to ECS cluster.
+
+If you're not familiar with ECS, you can get examples for those files from [here](/docs/examples/#ecs-applications).
+
+## Quick sync
+
+By default, when the [pipeline](/docs/user-guide/configuration-reference/#ecs-application) was not specified, PipeCD triggers a quick sync deployment for the merged pull request.
+Quick sync for a ECS deployment will roll out the new version and switch all traffic to it.
```

Comment:
nit: "an ECS"

Reply:
nit: "switch all traffic to it immediately."

Reply:
Addressed on https://github.com/pipe-cd/pipe/pull/2080/commits/b5f3e68d674fc950bd733abcf071688f01034a07 🙏